web/src/TopBar.tsx

```diff
@@ -46,33 +40,12 @@ class TopBar extends PureComponent<TopBarProps> {
   }
 }
 
-function renderSnapshotLinkButton(
-  snapshot: Snapshot,
-  handleSendSnapshot: (snapshot: Snapshot) => void,
-  snapshotURL: string
-) {
-  let hasLink = snapshotURL !== ""
-  if (!hasLink) {
-    return (
-      <section className="TopBar-snapshotUrlWrap">
-        <button onClick={() => handleSendSnapshot(snapshot)}>
-          Share Snapshot
-        </button>
-      </section>
-    )
-  } else {
-    return (
-      <section className="TopBar-snapshotUrlWrap">
-        <p className="TopBar-snapshotUrl">{snapshotURL}</p>
-        <button
-          title="Open link in new tab"
-          onClick={() => window.open(snapshotURL)}
-        >
-          Open
-        </button>
-      </section>
-    )
-  }
+const renderSnapshotModal = (handleOpenModal: () => void) => {
```

Comment:
this would make more sense as a method of TopBar? that way it has access to this.props. It creates a sub-element of topbar and doesn't seem generally useful outside it